lib/entities/organization-membership.js

```diff
@@ -0,0 +1,126 @@
+/**
+ * Organization membership instances
+ * @namespace OrganizationMembership
+ */
+
+import cloneDeep from 'lodash/cloneDeep'
+import { freezeSys, toPlainObject } from 'contentful-sdk-core'
+import enhanceWithMethods from '../enhance-with-methods'
+import errorHandler from '../error-handler'
+
+/**
+ * @memberof OrganizationMembership
+ * @typedef OrganizationMembership
+ * @prop {Meta.Sys} sys - System metadata
+ * @prop {string} role
+ * @prop {boolean} status - User is an admin
+ * @prop {function(): Object} toPlainObject() - Returns this Organization Membership as a plain JS object
+ */
+
+/**
+ * @memberof OrganizationMembership
+ * @typedef OrganizationMembershipCollection
+ * @prop {number} total
+ * @prop {number} limit
+ * @prop {number} skip
+ * @prop {Object<{type: "Array"}>} sys
+ * @prop {Array<OrganizationMembership.OrganizationMembership>} items
+ * @prop {function(): Object} toPlainObject() - Returns the collection as a plain JS object
+ */
+
+function createOrganizationMembershipApi (http, orgId) {
+  const baseURL = http.defaults.baseURL.replace('/spaces/', `/organizations/${orgId}`)
+
+  return {
+
+    /**
+     * Sends an update to the server with any changes made to the object's properties
+     * @memberof OrganizationMembership
+     * @func update
+     * @return {Promise<OrganizationMembership>} Object returned from the server with updated changes.
+     * @example
+     * const contentful = require('contentful-management')
+     *
+     * const client = contentful.createClient({
+     *   accessToken: '<content_management_api_key>'
+     * })
+     *
+     * client.getOrganizations()
+     * .then((organizations) => organizations.items[0])
+     * .then(org => org.getOrganizationMembership('organizationMembership_id'))
+     * .then((organizationMembership) => {
+     *  organizationMembership.role = 'member';
+     *  organizationMembership.update();
+     * })
+     * .catch(console.error)
+     */
+
+    update: function () {
+      const raw = this.toPlainObject()
+      const { role } = raw
```

Comment:
@phoebeschmidt maybe this, instead of deleting everything. wdyt?

Reply:
nice! yes better :)